File: week04/week04-1.cpp
//week04-1.cpp  C++ セ
//LeetCode 2529.Maximum Count of Positive Integer and Negative Integer
class Solution {
public:
    int maximumCount(vector<int>& nums) {
        int pos=0,neg=0;
        for(int i=0;i<nums.size();i++){
            if(nums[i]>0)pos++;//タ计++
            if(nums[i]<0)neg++;//璽计++

        }
        if (pos>neg) return pos;//タ计ゑ耕 癳
        else return neg;//ぃ礛璽计ゑ耕 癳
    }
};

File: week04/week04-1.py
# week04-1.py 二合一的python版本
# LeetCode
class Solution:
    def maximumCount(self, nums: List[int]) -> int:
        pos = 0
        neg = 0
        for i in range(len(nums)):
            if nums[i] > 0: pos+= 1 #正數
            if nums[i] < 0: neg+= 1 #負數

        return max(pos,neg) #把大的數return傳回去
